src/components/accounts/create/ActivateAccount.js

```diff
@@ -241,7 +246,7 @@ class ActivateAccount extends Component {
 const mapStateToProps = ({ account, status }) => ({
     ...account,
     mainLoader: status.mainLoader,
-    minBalanceToUnlock: account.accountHelperWalletState?.requiredUnlockBalance,
+    minBalanceToUnlock: account.accountHelperWalletState?.requiredUnlockBalance || parseNearAmount(WALLET_APP_MIN_AMOUNT),
```

Comment:
Lets make sure we follow up and move this into a selector co-located with the `accountHelperWalletState` slice